### DISCUSSION TITLE
Malfunction / problem ep_patcher

### DISCUSSION THREAD

**Discussion by RosteWezel** [valinet/ExplorerPatcher#746] [Q&A]
Hi

I am not paticularly well-versed in pc's so forgive if I do not offer an adequate amount of information.

When I downloaded & installed the ep_patcher my taskbar dissapeared together with the whole desktop background and all the apps that were on my main screen. It is now completely black.

My cursor is still visible, but everything else does not seem to work (e.g. my windows key, adjusting volume & brightness...)

The only thing which seems to work is control alt delete

I can't deinstall the patcher because of this lack of functions.

Any fix?

Thanks in advance, I'll provide as much info as possible...

**Comment by RosteWezel** [comment]
Was able to reach file through task manager and uninstalled.

**Comment by Dittotoo** [comment]
See the following discussion thread for similar problem and reply.

https://github.com/valinet/ExplorerPatcher/discussions/729
